packages/checkout-common/rollup.config.js

```diff
@@ -0,0 +1,33 @@
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import typescript from "@rollup/plugin-typescript";
+import { terser } from "rollup-plugin-terser";
+import external from "rollup-plugin-peer-deps-external";
+import dts from "rollup-plugin-dts";
+import esbuild from "rollup-plugin-esbuild";
+
+const packageJson = require("./package.json");
+
+export default [
+  {
+    input: "src/index.ts",
+    output: [
+      {
+        file: packageJson.main,
+        format: "cjs",
+        sourcemap: true,
+      },
+      {
+        file: packageJson.module,
+        format: "es",
+        sourcemap: true,
+      },
+    ],
+    plugins: [esbuild(), typescript({ tsconfig: "./tsconfig.json" })],
```

Comment:
Why are we not using the same rollup config as other packages in this repo?

Reply:
Honestly: I've used it when I firstly tried to debug why the build was failing, and I couldn't get it to work, the erorr messages were too cryptic for me 😅.

I've tried simpler config, just cjs and esbuild and it started to work for me. I've left it since I though that maybe we should use esbuild rather than multiple plugins? We already use it to build dashboard and macaw-ui. The performance difference is massive, although for this small package this doesn't even matter since we literaly export 1 variable and the rest is just types.

I've copied config from ui-kit `-` postcss